elixir/apps/web/lib/web/components/core_components.ex

```diff
@@ -852,13 +852,15 @@ defmodule Web.CoreComponents do
   attr :rows, :list, required: true
   attr :row_id, :any, default: nil, doc: "the function for generating the row id"
   attr :row_click, :any, default: nil, doc: "the function for handling phx-click on each row"
+  attr :subject, :string, required: false
```

Comment:
I don't think this attribute is used here, and it should not be passed here as it exposes raw auth data to the view logic.

Reply:
So, it isn't required, but in order to render almost all of the the `navigate` URLs in the table cells, it needs to have the `subject`.  Also, I was getting a warning in my IDE tooling without having it here since I was passing the `subject` in as an attribute when calling `<.table>`

Is there a better alternative than passing this in through the component attributes?

Reply:
Replied in another thread below

Reply:
Removed this as discussed in another thread below